sopel/modules/tell.py

```diff
@@ -122,16 +143,18 @@ def f_remind(bot, trigger):
     if tellee not in (Identifier(teller), bot.nick, 'me'):
         tz = get_timezone(bot.db, bot.config, None, tellee)
         timenow = format_time(bot.db, bot.config, tz, tellee)
+        timestamp = datetime.datetime.timestamp(datetime.datetime.now())
+        timestamp = str(int(timestamp))
         bot.memory['tell_lock'].acquire()
         try:
             if tellee not in bot.memory['reminders']:
-                bot.memory['reminders'][tellee] = [(teller, verb, timenow, msg)]
+                bot.memory['reminders'][tellee] = [(teller, verb, timenow, timestamp, msg)]
             else:
-                bot.memory['reminders'][tellee].append((teller, verb, timenow, msg))
+                bot.memory['reminders'][tellee].append((teller, verb, timenow, timestamp, msg))
```

Comment:
Refactoring opportunity: Use the `if thing not in memory: initialize it` pattern to avoid duplicating the tuple format. Pseudocode:

```
if tellee not in reminder-memory:
    reminder-memory[tellee] = []
reminder-memory[tellee].append((reminder, data, tuple))
```